ios/FluentUI/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -96,6 +93,9 @@
 /* Accessibility label format string for avatar view. Format: "<Name>, <Presence>". Example: "Kat, Available" */
 "Accessibility.AvatarView.LabelFormat" = "%@, %@";
 
+/* Accessibility label for when content is loading */
+"Accessibility.Shimmer.Loading" = "Content Loading";
```

Comment:
Does this require a localization pass?

Reply:
I think localization automatically happens after I check in